hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -489,10 +490,11 @@ public ContainerBackgroundTaskResult deleteViaTransactionStore(
               deletedBlocksCount, releasedBytes);
 
           // update count of pending deletion blocks, block count and used
-          // bytes in in-memory container status.
+          // bytes in in-memory container status and used space in volume.
           containerData.decrPendingDeletionBlocks(deletedBlocksCount);
           containerData.decrBlockCount(deletedBlocksCount);
```

Comment:
Isn't it a good idea to decrement the space in containerData.decrBytesUsed ?
So, that increment and decrement will be in same place?

Reply:
But for increment we did inside containerData even for volume. I am wondering if we do all inside container data may be consistent? ( Unless we have any other potential issues in doing that.)

Reply:
For increment we had `incrWriteBytes()` which is used in `updateWriteStats()` and is different from `incrBytesUsed()`, both are done as two operations inside `updateWriteStats()`. We can do something like create a function `decrWriteBytes()` and then call `decrementUsedSpace()`, but it will be same instead of calling `containerData.getVolume().decrementUsedSpace()` we will be calling `containerData.decrWriteBytes()`